nodejs/week1/week1homework/src/bakckend/routes/cheap-meals.js

```diff
@@ -0,0 +1,18 @@
+function getCheapMeals(){
+    
+    const mealArr=require("../data/meals.json");
+    const reviewArr=require("../data/reviews.json");
+
+    const reviewMeals = mealArr.map((meal)=>{
+        meal.reviews = reviewArr.filter((review) => review.mealId === meal.id);
+        return meal;
+    })
+    //assuming cheap meals are with price less than 100 kr
+    const cheapMeals = reviewMeals.filter((meal) =>  meal.price <= 100);
```

Comment:
This code is duplicated across several files. When you find yourself writing the same code more than twice, consider whether you can extract a helper function that you can reuse from both places. This principle is commonly known as DRY (Don't Repeat Yourself). Keeping your code DRY helps you right more maintainable code because:

* If there's a bug in this code, it's only in one place and you only need to fix it once
* When you create a helper function other team mates can benefit from it and reuse your code
* When someone looks at your code they need to read it slowly and understand what is going on. When they read a helper function like getRandomInteger(maxValue) they automatically understand what the function does without needing to exert their cognitive abilities a lot.

Reply:
Consider creating a `getMealsWithReviews` function, put it in its own module and reuse it from other routers.